cmd/tnf/generate/catalog/catalog.go

```diff
@@ -32,6 +33,13 @@ import (
 )
 
 var (
+	feedbackjson string
+	// generateCmd is the root of the "catalog generate" CLI program.
+	generatefeedbackjs = &cobra.Command{
+		Use:   "feedbackjs",
+		Short: "Generates feedbackjs from feedbackjson file and copy it into /cnf-certification-test/results/html",
```

Comment:
```suggestion
		Short: "Generates a javascript file called feedback.js from a feedback.json that was downloaded from the results html viewer",
```
By the way, the flags "Long" and "Example" flags can also be used to include more detailed info about both files and some examples.